### ISSUE TITLE
Save image embedded in maps

### ISSUE THREAD

**Issue by zoga-com** [MCCTeam/Minecraft-Console-Client#1597]
Welcome. Is it possible to somehow pull the image from the card in your hands? I just would like to set up a log bot for logging messages from the chat, but I ran into a problem that asks me to enter a captcha at the entrance, which I simply can't do, because I don't know how to just get an image. Immediately I say-the server is not mine, which is why the answer "remove the bot filter" is not accepted.

**Comment by ReinforceZwei** [comment]
No, it is not possible. MCC is a console application and it can't display image.

**Comment by kungermu** [comment]
But maybe Inventory Handling have this function?

**Comment by kungermu** [comment]
Just get (download) image from map, no more.

**Comment by ReinforceZwei** [comment]
> But maybe Inventory Handling have this function?

Unfortunately no. Map item only has map ID stored in it. Retrieving actual map data isn't handled.

**Comment by ORelio** [comment]
Converting this question into a feature request.

**Comment by kungermu** [comment]
Thank. I wait. :-)

**Comment by milutinke** [comment]
Should be possible now after #2189 has been merged.
Looking at creating a ChatBot to handle this.

**Comment by milutinke** [comment]
![image](https://user-images.githubusercontent.com/441903/192608288-af70a638-866e-4a70-8791-ab894989b49a.png)
Getting there slowly

**Comment by milutinke** [comment]
In action
![image](https://user-images.githubusercontent.com/441903/192611173-16494a48-0da4-4473-aa75-3c736daa0d1b.png)

**Comment by milutinke** [comment]
Implemented in #2215

**Comment by nnaskz** [comment]
> Как сделать, чтобы капча грузилась полностью, а не по кусочкам помогите сервер holyworld ![Скриншот 01-01-2024 164533](https://private-user-images.githubusercontent.com/109684141/293649694-d3776234-0593-4619-8631-fa1f0423a2b8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxMzk5MDgsIm5iZiI6MTcyMjEzOTYwOCwicGF0aCI6Ii8xMDk2ODQxNDEvMjkzNjQ5Njk0LWQzNzc2MjM0LTA1OTMtNDYxOS04NjMxLWZhMWYwNDIzYTJiOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOFQwNDA2NDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMGNiMzM5ZWUzOTBmY2EyNWQ2NGUzNGY3YjlmM2IwYTlkZDNhYzdhM2I1OGY1ZGY2YWNhOThkZTU5MTY4YzhkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.fKmNW59vOhpScpsOPTheqTpscBddfI2vuk3aXyxGm2g)

разобрался?

**Comment by breadbyte** [comment]
Building the maps into one singular image will not be implemented as discussed in issue https://github.com/MCCTeam/Minecraft-Console-Client/issues/2249

For any other issues relating to this feature, please create a new issue or discussion.
